art/utils.py

```diff
@@ -1212,3 +1212,17 @@ def is_probability(vector: np.ndarray) -> bool:
     is_larger_0 = np.amin(vector) >= 0.0
 
     return is_sum_1 and is_smaller_1 and is_larger_0
+
+
+def pad_audio_input(x: np.ndarray) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
```

Comment:
Is the padding specific to audio or could it be applied to any sequence data? I'm wondering if we should name this function more generally e.g. replace audio with sequence as it could have wider use beyond audio.
```suggestion
def pad_audio_input(x: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
```

Reply:
It could be applied to any batch of 1-dimensional sequence data. I have followed your suggestion and renamed the function. I also improved the docstring.